cmd/docker/docker.go

```diff
@@ -298,6 +304,15 @@ func runDocker(dockerCli *command.DockerCli) error {
 		return err
 	}
 
+	mp := dockerCli.MeterProvider(ctx)
+	defer mp.Shutdown(ctx)
+	otel.SetMeterProvider(mp)
+	meter := mp.Meter(
+		"github.com/docker/cli",
+		metric.WithInstrumentationVersion(cmd.Version),
```

Comment:
https://github.com/docker/buildx/blob/fe08cf2981836f1828058f02cb7684f676b53a9e/util/metricutil/metric.go#L216-L219 looks like we should use the CLI version here.

Reply:
```suggestion
		metric.WithInstrumentationVersion(version.Version),
```

Reply:
Oh! I totally missed that we have a version.go as well, thanks for spotting :)